internal/namespaces/lb/v1/custom_backend.go

```diff
@@ -21,3 +21,48 @@ var (
 		lb.BackendServerStatsServerStateStopping: &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "stopping"},
 	}
 )
+
+func lbMarshalerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
+	type tmp lb.LB
+	loadbalancer := tmp(i.(lb.LB))
+
+	opt.Sections = []*human.MarshalSection{
+		{
+			FieldName: "IP",
+		},
+		{
+			FieldName: "Instances",
```

Comment:
Not necessarily an instance. Could be a baremetal or even something completely different

Reply:
Why not Backends?

Reply:
Also why is Frontend missing?